locale/update_translations.sh

```diff
@@ -28,13 +17,12 @@ command -v tx >/dev/null 2>&1 || { echo >&2 "tx (transifex-client) is needed but
 
 # check if working directory is clean to ensure we only commit localization changes
 if test 0 -ne `git status -s -uno |wc -l`; then
-  echo "Please commit your pending changes first"
-  exit 1
+  echo "Warning: You have pending changes! Please make sure to only commit localization changes!"
```

Comment:
Did you intentionally remove the exit if there are pending changes?

Reply:
Yes, although in hindsight I should have added `read -rsp $'Press enter to continue...\n'` to make sure the user of the script sees the message. Otherwise it might vanish too soon. I might add that if no other problems arise.

Reply:
Ok - the script doesn't automatically commit like it used to so this is probably a nice to have rather than a must.  I'm satisified with this code.